src/rgw/rgw_op.h

```diff
@@ -1838,6 +1840,7 @@ class RGWInitMultipart : public RGWOp {
   RGWInitMultipart() {}
 
   void init(rgw::sal::Store* store, struct req_state *s, RGWHandler *h) override {
+    multipart_trace = tracing::rgw::tracer.start_trace(name());
```

Comment:
is the trace going to have the same name as the generic op name in this case?
maybe we should name it differently, so it is less confusing for the user?